amazonlambda/samples/amazonlambda/lambda_test.bal

```diff
@@ -0,0 +1,95 @@
+import org.wso2.ballerina.connectors.amazonlambda;
+import ballerina.lang.jsons;
+import ballerina.lang.messages;
+import ballerina.lang.system;
+import ballerina.net.http;
+import ballerina.test;
+
+message lambdaResponse;
+json lambdaJSONResponse;
+
+function init()(amazonlambda:ClientConnector){
+	
+	string accessKeyId = system:getEnv("ACCESS_KEY");
+        string secretAccessKey = system:getEnv("SECRET_ACCESS_KEY");
+        string region = system:getEnv("REGION");
+	amazonlambda:ClientConnector lambdaConnector = create amazonlambda:ClientConnector(accessKeyId, secretAccessKey,region);
+ 	return lambdaConnector;
+}
+
+function testInvokeFunction(){
+	amazonlambda:ClientConnector lambdaConnector = init();
+	lambdaResponse = amazonlambda:ClientConnector.invokeFunction(lambdaConnector, "BallerinaTestFunction");
+        lambdaJSONResponse = messages:getJsonPayload(lambdaResponse);
+        system:println(jsons:toString(lambdaJSONResponse));
+	int status = http:getStatusCode(lambdaResponse);
+        test:assertEquals(status, 200);
+        system:println("===testInvokeFunction completed===\n");	
+}
+
+
+function testInvokeFunctionWithParam(){
+        amazonlambda:ClientConnector lambdaConnector = init();
+	json payload = {"name":"Ballerina"};
+        lambdaResponse = amazonlambda:ClientConnector.invokeFunction(lambdaConnector, "BallerinaTestFunction", payload);
+        lambdaJSONResponse = messages:getJsonPayload(lambdaResponse);
+        system:println(jsons:toString(lambdaJSONResponse));
```

Comment:
we don't need to do jsons:toString anymore, actually those helper methods will get removed soon, just do system:println(json);